benchopt/benchmark.py

```diff
@@ -277,6 +278,14 @@ def validate_solver_patterns(self, solver_patterns):
 
         _validate_patterns(all_solvers, solver_patterns, name_type='solver')
 
+    def validate_objective_filters(self, objective_params):
+        "Check that all objective filters match at least one objective setup"
+
+        # List all choices of objective parameters
+        all_params = _list_all_objective_params(self.get_benchmark_objective())
```

Comment:
You should simply use `_list_all_parametrized_names`. No need for an extra function here.

Reply:
I would call this `all_objectives` to be consistent with the previous functions.